### DISCUSSION TITLE
Add rewind and fast forward to media controls in USBHIDConsumerControl.h

### DISCUSSION THREAD

**Discussion by jmcdeg** [espressif/arduino-esp32#10585] [Ideas]
I created a wireless windows media controller on a S2 mini. All works well to play/pause music, and to skip tracks etc. It took me a while though to find the codes for rewind and fast forward, which I like to share. I suggest to add CONSUMER_CONTROL_REWIND (0x00B4) and CONSUMER_CONTROL_FAST_FORWARD (0x00B3) to USBHIDConsumerControl.h.

**Comment by SuGlider** [comment]
Added as Feature Request.

**Reply by SuGlider** [reply]
PR #11227 adds those and a few more. Thanks!

**Reply by jmcdeg** [reply]
AWESOME, thanks!!!

**Comment by RichiG88** [comment]
If you need some more codes, you can find them here:
https://usb.org/sites/default/files/hut1_4.pdf#page126
